sample-apps/java/client/devserver-client-sample/src/main/java/clientsample/YourWorker.java

```diff
@@ -0,0 +1,29 @@
+package clientsample;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.worker.Worker;
+import io.temporal.worker.WorkerFactory;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+
+public class YourWorker {
+    public static void initiateWorker(String[] args) {
+        // Create a stub that accesses a Temporal Service on the local development machine
+        WorkflowServiceStubs service = WorkflowServiceStubs.newLocalServiceStubs();
+
+        // A Workflow Client can start, Signal, and Query a Workflow Execution, etc
+        WorkflowClient client = WorkflowClient.newInstance(service);
+        
+        // A Workflow Factory creates Workers
+        WorkerFactory factory = WorkerFactory.newInstance(client);
+        
+        // A Worker listens to one task queue, processing Workflows and Activities
```

Comment:
Workers can be configured to process _only_ Workflow (or _only_ Activities), so I suggest a minor rephrasing to be clear that we're talking about what this particular one does.

```suggestion
        // A Worker listens to one task queue. This Worker will process both Workflows and Activities
```